.github/workflows/pr-checks.yml

```
@@ -19,4 +19,4 @@ jobs:
     with:
       botName: Eclipse Equinox Bot
       botMail: equinox-bot@eclipse.org
-      extra-maven-args: '-f bundles'
+      extra-maven-args: '-Dequinox.binaries.loc=$(pwd)/binaries -Pfetch-natives'
```

An alternative to the new profile would be to use https://github.com/eclipse-platform/eclipse.platform.releng.aggregator/pull/2783 and just add:
```suggestion
      extra-setup-command: 'git clone --depth 1 https://github.com/eclipse-equinox/equinox.binaries.git binaries'
      extra-maven-args: '-Dequinox.binaries.loc=$(pwd)/binaries -Pfetch-natives'
```
This would avoid all changes in the pom.xml files and would be much more compact.
On the other hand, having an 'internal' definition of the binaries-fetch would allow to re-use it in other workflows and even locally, which would simplify local builds as it avoids the need to clone the binaries repo in advance.